src/scripts/deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+ORB_EVAL_APPLICATION_NAME=$(eval echo "${ORB_EVAL_APPLICATION_NAME}")
```

Comment:
We have `env subst` coming soon, because we know this causes some breaking changes for some, we should maybe wait until env subst is deployed

Reply:
My thought was to merge all these changes into `master` first and when `env subst` is available, we can cut a new branch and add it before cutting a new major release. That's been my assumption for all the other orbs as well. 

Let me know what you think.